### ISSUE TITLE
Can't save a monster's alternate evolution

### ISSUE THREAD

**Issue by theinfra** [madcowfred/padherder-issues#347]
When I save a monster that's already in an ult evo form (say Inferno Deity Falcon, Horus) and I wan't to save the alt evo (Blazing Deity Falcon, Horus) to track the materials needed for the evolution,
- I see the "1 alternate evolution available" text
- Click Edit
- Choose the alternate evolution
- Click save
- Any other changes get save (like xp, +eggs, notes, etc) but the alt evo doesn't and the text "1 alternate..." stays

I tried this with many monsters and different priorities. I already had my monsters with alt evos chosen but then I re-arranged the priorities and the alt evos won't save in any of them.

In mobile mode (adding "m." before the url) it does work.

**Comment by theinfra** [comment]
Sorry to bump, but this issue is still present and I don't think I'm doing anything wrong.

Another thing I discovered today, is that the monsters that I can't save with alternate evolution, this alternate evolution is not shown in the Mobile version.

For example, I have Noble Wolf King Hero, Cu Chulainn (#693) and I wish to save with the Desired Evolution to Wolf Hero, Ignis Cu Chulainn (#1209), hoping that in the Monsters table I would have the materials needed to reverse the Ulti Evo (1 lit of each and then the evo for Ignis).

There are 2 issues here andf maybe they are related:
- In the desktop version, I select Ignis, press save but it's not saved, just like I wrote above.
- In the mobile version, in the Edit screen, in the Evolution section, I cannot see Ignis as a possible alternative evolution to Cu Chu, so I cannot save the evo either
